### PR TITLE
feat: change klaytn to kaia chain

### PR DESCRIPTION
**Pull request by 0oooooooo0**
This Pull Request proposes the rebranding of the Klaytn Chain to the Kaia Chain, effective from August 29th. The changes include the following:

All instances of "Klaytn" have been updated to "Kaia" throughout the codebase and documentation.
The KlaytnBaobab testnet has been renamed to "Kairos." All references to "KlaytnBaobab" have been updated accordingly.
Please review the changes and provide feedback. If everything is in order, I would appreciate it if this PR could be merged by or before August 29th to ensure a smooth transition.

For more details about the chain rebranding, please refer to the official announcement [here](https://medium.com/kaiachain/its-official-kaia-mainnet-will-be-going-live-on-29-august-42fc10ea9583).

---

## PR-Codex overview
The focus of this PR is to add support for the `Kaia` chain and deprecate `klaytn` in favor of `kaia` and `kairos`.

### Detailed summary
- Added `Kaia` chain with details
- Deprecated `klaytn`, use `kaia` instead
- Added `kairos` chain
- Updated `klaytn` and `klaytnBaobab` chains with new details

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/chains/definitions/kaia.ts [hunk 1]
```diff
@@ -1,20 +1,20 @@
 import { defineChain } from '../../utils/chain/defineChain.js'
 
-export const klaytn = /*#__PURE__*/ defineChain({
+export const kaia = /*#__PURE__*/ defineChain({
```
Comment (jxom) [inline src/chains/definitions/kaia.ts:3]:
```suggestion
export const klaytn = /*#__PURE__*/ defineChain({
```

File: src/chains/definitions/kairos.ts [hunk 2]
```diff
@@ -1,21 +1,21 @@
 import { defineChain } from '../../utils/chain/defineChain.js'
 
-export const klaytnBaobab = /*#__PURE__*/ defineChain({
+export const kairos = /*#__PURE__*/ defineChain({
```
Comment (jxom) [inline src/chains/definitions/kairos.ts:3]:
```suggestion
export const klaytnBaobab = /*#__PURE__*/ defineChain({
```

**Review by jxom — CHANGES_REQUESTED** [review]
To avoid breaking changes, can you please keep the original exports for these chains under a `/** @deprecated */` flag and also add new files for the new chains? We can keep the rebranding in the old chains too.

Example:
https://github.com/wevm/viem/blob/main/src/chains/index.ts#L226-L227

**Comment by 0oooooooo0** [comment]
> To avoid breaking changes, can you please keep the original exports for these chains under a `/** @deprecated */` flag and also add new files for the new chains? We can keep the rebranding in the old chains too.
> 
> Example: https://github.com/wevm/viem/blob/main/src/chains/index.ts#L226-L227

I completed it! thank you

**Comment by 0oooooooo0** [comment]
@jxom Do you check it? Kaia will launch after 2 days. so I hope it merges quickly

**Pull request merged**